src/experimental/DataBase.h

```diff
@@ -76,7 +77,22 @@ class DataBase {
      */
     TableData getEntryData(DataPosition);
 
-    TableData getEntryDataNew(DataPosition);
+    /*!
+     * \brief retreive an Entry from the database.
+     */
+    Entry getEntry(DataPosition);
+
+    /*!
+     * \brief retreives a PilotEntry from the database.
+     *
+     * This function is a wrapper for DataBase::getEntry(DataPosition),
+     * where the table is already set and which returns a PilotEntry
+     * instead of an Entry. It allows for easy access to a pilot entry
+     * with only the RowId required as input.
+     */
+    PilotEntry getPilotEntry(RowId);
+    ///[F] the same can easily be implemented for tails/flights
```

Comment:
Im sure we will see code duplication here. But lets write it first before cleaning it up so that we are sure we are on the right path.